# Module13: Analyzing RNA-seq data in Saccharomyces cerevisiae
The main purpose of these exercises is to learn how to use command-line based tools to analyze gene expression (RNA-seq) data in yeast (Saccharomyces cerevisiae). This eukaryotic organism is used here du the its relatively small genome, meaning that the data set we use will be easy to handle quickly. We will download the sequence data, the reference genome sequence and the genome annotation. We will use a tool called STAR for mapping, and this time we will use a desktop browser called IGV for visualizing the results. Since the programs we are using are designed to be run on the terminal, as is typically the case for large-scale bioinformatics programs, we will once more utilize the Saga system.


The main learning outcomes of this exercise are:
- Using an external server to run a command-line pipeline starting from the raw
sequence to final results
- Using and loading tools available on the server
- Downloading and uploading data to/from the server
- Performing a simple gene expression analysis with RNA-seq data
- Visualizing and exploring our results in the Integrative Genome Browser (IGV)

## 13.1 Mapping RNA-seq reads to an indexed reference genome sequence
- Go to [these exercises](exercises/Exercise1.md) and follow the steps
## 13.2 Visualizing mapped reads in a genome browser (IGV)
- Go to [these exercises](exercises/Exercise2.md) and follow the steps

